lib/apartment.rb

```diff
@@ -0,0 +1,28 @@
+class Apartment
+  attr_reader :renter
+
+  def initialize(info_hash)
+    @info_hash = info_hash
```

Comment:
Breaking this out and storing in separate instance variables allows us to just leverage attr_reader